bin/fmt

```diff
@@ -5,8 +5,13 @@
 source bin/lib.sh
 docker::set_project_name_dev
 
-bin/pull-image
+if [[ -z "${USE_LOCAL_CIVIFORM}" ]]; then
```

Comment:
I don't love pulling this out from the main pull-image.  Perhaps the bin/pull-image script can take an optional argument of the image name (and pull all if not provided)

Reply:
done